client/packages/common/src/intl/locales/en/distribution.json

```
@@ -70,6 +70,7 @@
   "label.units_other": "units",
   "label.units-in-pack-size-of_one": "unit in packs of",
   "label.units-in-pack-size-of_other": "units in packs of",
+  "message.placeholder-line": "Placeholder line has been created for unallocated line.",
```

:nit better without the full stop imho
:nit
```suggestion
  "message.placeholder-line": "A placeholder line has been created for additional requested quantity",
```